host/tests/dsHost_L3_Tests/dsHost_test1_VerifyTemperature.py

```diff
@@ -0,0 +1,156 @@
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../")
+
+from dsClasses.dsHost import dsHostClass
+from raft.framework.plugins.ut_raft import utHelperClass
+from raft.framework.plugins.ut_raft.configRead import ConfigRead
+from raft.framework.plugins.ut_raft.utPlayer import utPlayer
+from raft.framework.plugins.ut_raft.utUserResponse import utUserResponse
+
+class dsHost_test1_EnableDisableAndVerifyAudioPortStatus(utHelperClass):
+
+    testName  = "test1_VerifyTemperature"
+    testSetupPath = dir_path + "/dsHost_L3_testSetup.yml"
+    moduleName = "dsHost"
+    rackDevice = "dut"
+
+    def __init__(self):
+        """
+        Initializes the test1_VerifyTemperature test .
+
+        Args:
+            None.
+        """
+        super().__init__(self.testName, '1')
+
+        # Test Setup configuration file
+        self.testSetup = ConfigRead(self.testSetupPath, self.moduleName)
+
+        # Open Session for player
+        self.player_session = self.dut.getConsoleSession("ssh_player")
+
+        # Open Session for hal test
+        self.hal_session = self.dut.getConsoleSession("ssh_hal_test")
+
+        player = self.cpe.get("test").get("player")
+
+        # Create player Class
+        self.testPlayer = utPlayer(self.player_session, player)
+
+         # Create user response Class
+        self.testUserResponse = utUserResponse()
+
+        # Get path to device profile file
+        self.deviceProfile = dir_path + "/" + self.cpe.get("test").get("profile")
+
+    def testDownloadAssets(self):
+        """
+        Downloads the artifacts and streams listed in test-setup configuration file to the dut.
+
+        Args:
+            None.
+        """
+
+        # List of streams with path
+        self.testStreams = []
+
+        self.deviceDownloadPath = self.cpe.get("target_directory")
+
+        #download test artifacts to device
+        url = self.testSetup.assets.device.test1_EnableDisableAndVerifyAudioPortStatus.artifacts
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+
+        #download test streams to device
```

Comment:
self.testStreams = [] is to store the stream paths.

url = self.testSetup.assets.device.test1_VerifyTemperature.artifacts 
this will get the artifacts of the test.

So self.testStreams = [] is to store the stream paths. is not required unless we are using streams in testing.

Reply:
Line 74, 84-88 is for stream download.

Reply:
Streams removed.